### PR TITLE
1144 store preferences

### LINKED ISSUE #1144: Store Preferences
Following up on the KDD created in #1087 

Option chosen: 
Create a store preference table with store_id as the primary key and have strongly typed columns for the preferences. Currently only need pack_to_one preference

### PR DESCRIPTION
**Pull request by roxy-dao**
Closes #1144 
- Updated DB version
- Created table for store preferences
- Sync table 

Should these preferences be shown anywhere...? Need api call for it? 🤔

### CODE REVIEW

File: server/repository/src/migrations/v1_01_03/mod.rs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+impl Migration for V1_01_03 {
+    fn version(&self) -> Version {
+        Version::from_str("1.1.3")
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        use crate::migrations::sql;
+
+        sql!(connection, r#"DROP VIEW IF EXISTS item_is_visible;"#)?;
```
Comment (roxy-dao) [inline server/repository/src/migrations/v1_01_03/mod.rs:14]: Moved this since it was done after 1-1-2 release

**Comment by mark-prins** [comment]
an API for the preferences would be great!
Another time though.. it isn't critical to this piece of work: https://github.com/openmsupply/open-msupply/issues/1169

**Review by mark-prins — CHANGES_REQUESTED** [review]
👍 Looking good! One change requested please - the postgres migrations need a small fix.
Other than that, works very nicely

**Review by mark-prins — APPROVED** [review]
😁  and woo! preferences!!

**Pull request merged**